cli/CMakeLists.txt

```diff
@@ -44,6 +44,20 @@ configure_file (
   "${CMAKE_CURRENT_BINARY_DIR}/CliConfig.h"
 )
 
+# Compile the protos for Single Node Server mode.
+if(ENABLE_SINGLE_NODE_SERVER)
+  # We will need some of the TMBs libraries
+  set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/third_party/src/tmb/cmake")
+
+  find_package(Grpc++ REQUIRED)
+  include_directories(${GRPCPLUSPLUS_INCLUDE_DIRS})
```

Comment:
Also not that because of CMAKE scoping, the module will not be added to the parent scope's module path:

`/cmakelists.txt`:
```
message("before module path: ${CMAKE_MODULE_PATH}")
add_subdirectory(cli)
message("after module path: ${CMAKE_MODULE_PATH}")
```
output:
```
before module path: /Users/cramja/workspace/quickstep/incubator-quickstep/cmake
after module path: /Users/cramja/workspace/quickstep/incubator-quickstep/cmake
```